xdsl/ir/core.py

```diff
@@ -517,14 +518,49 @@ def __hash__(self) -> int:
         ...
 
 
+@dataclass
+class OpOperands(Sequence[SSAValue]):
+    """
+    A view on an operation operand list.
```

Comment:
```suggestion
    A view of the operand list of an operation.
```
Sounds a bit less confusing :wink: